bin/getInfoRippled.sh

```diff
@@ -1,41 +1,72 @@
 #!/usr/bin/env bash
 
-TMP_LOC=$(mktemp -d /tmp/ripple_info.XXXX)
-echo ${TMP_LOC}
+rippled_exe=/opt/ripple/bin/rippled
+conf_file=/etc/opt/ripple/rippled.cfg
 
-cd ${TMP_LOC}
+while getopts ":e:c:" opt; do
+    case $opt in
+        e)
+            rippled_exe=${OPTARG}
+            ;;
+        c)
+            conf_file=${OPTARG}
+            ;;
+        \?)
+            echo "Invalid option: -$OPTARG"
+    esac
+done
 
-# Send output from this script to a log file
-## this captures any messages
-## or errors from the script itself
+tmp_loc=$(mktemp -d /tmp/ripple_info.XXXX)
+cd /tmp
+chmod 751 ripple_info.*
+cd ~
```

Comment:
I added this because the instructions to scp the folder out of the instance won't work with the permissions set by mktemp - which I realized wasn't in this version of the script - will add in the next version: 

echo "Use the following command on your local machine to download from your rippled instance: scp <remote_rippled_username>@<remote_host>:${tmp_loc}/info-package.tar.gz <path/to/local_machine/directory>"| tee /dev/fd/3